conf/messages

```diff
@@ -19,6 +19,15 @@ businessType.other.signOut = Finish and sign out
 businessType.error.required = Tell us how your business is set up
 businessType.error.invalid = Tell us how your business is set up
 
+# About your application / user role
+userRole.title = Are you the owner of the business?
+userRole.true = Yes
+userRole.false = No, but I''m authorised by them to set up this account
+userRole.other.signOut = Finish and sign out
+userRole.error.required = Tell us how your business is set up
+userRole.error.invalid = Tell us how your business is set up
```

Comment:
sorry only noticed it was the wrong error message when I saw your new test that asserts it... the content from the ticket was `Select ‘yes’ if you are the owner of the business`